code/datums/components/storage/storage.dm

```diff
@@ -614,6 +614,12 @@
 		if(!stop_messages)
 			to_chat(M, "<span class='warning'>[I] is too big for [host]!</span>")
 		return FALSE
+	if(real_location.loc.GetComponent(/datum/component/storage))//return false if we are inside of another container,
+		var/datum/component/storage/biggerfish = real_location.loc.GetComponent(/datum/component/storage)

```

Comment:
No need to query this twice. Just check if this variable is null

Reply:
alrighty